js/components/operatorSignIn/text.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from "react";
+
+// 🚨 You MUST change the version if you change the text!
+// 🚨 You MUST update the document at https://www.notion.so/mbta-downtown-crossing/Attestation-Text-History-7f7117d7c4574013a48c6819c7809c4a?pvs=4
+export const useSignInText = (): {
+  version: number;
+  text: ReactElement;
+} => {
+  // Check the time only once on mount so that the text doesn't change if the time changes.
+  return {
+    version: 5,
```

Comment:
**question:** should start a separate version numbering scheme from 1 for Orbit?

Reply:
Yeah, we probably should! ed0fb290b3773c48936044a71eec21ef727c9dd5